src/stylesheets/uswds.scss

```diff
@@ -36,9 +36,9 @@ Output all USWDS
 @import 'elements/embed';
 @import 'elements/figure';
 @import 'elements/inputs';
-@import 'elements/labels';
 @import 'elements/list';
 @import 'elements/table';
+@import 'elements/tags';
```

Comment:
🎉🎉🎉 
Now, it might make sense to make a note of the breaking markup changes so we can include them in the migration guide

Reply:
Yeah, we should add any notable changes to 2.0 in https://github.com/uswds/uswds/releases/tag/v2.0.0-alpha